### PR TITLE
fix to cehck SUCEEDED OR FAILED at the end of code, also store ssh conne...

### PR DESCRIPTION
**Pull request by k-okada**
...ction

### CODE REVIEW

File: hironx_ros_bridge/robot/robot-system-check.sh [hunk 1]
```diff
@@ -46,15 +46,29 @@ ping -c1 $hostname || (echo -e "-- [ERROR] Could not connect to $hostname"; exit
...
-ssh  $userid@$hostname "touch /opt/jsk/.checked"
+ssh -o 'ControlPath=~/.ssh/%r@%h:%p.conn'  $userid@$hostname -t $commands 2>&1 | tee robot-system-check-$hostname.log
+RESULT=${PIPESTATUS[0]}
+ssh -o 'ControlPath=~/.ssh/%r@%h:%p.conn' -O exit "$userid@$hostname"
 
+echo $RESULT
 echo -e ";;\n;;\n;; Done check scripts, please check robot-system-check-$hostname.log file\n;;\n;;\n"
 
+if [ $RESULT == 0 ]; then
+    echo -e ";; RESULT: SUCEEDED"
```
Comment (130s) [inline hironx_ros_bridge/robot/robot-system-check.sh]: typo `succeed`

**Comment by 130s** [comment]
The idea is great; to judge the result only by true or false at the end (instead by the printed string). I ran it and the result was `success` while the printed string is false. Is it a bug or intended?

```
* Check /opt/nextage-open directories
  Check  /opt/nextage-open      ( 0x1fec7abb837fb5b412057e18061f75aaL ) False
  Writing results to ...  /tmp/tmpiRAp0d/localhost-20140708-230732/md5sum_opt_nextage_open.py
* Check /var files
* Check /usr/local files
* Check /usr/pkg files
* Check /opt files
 ** /opt/nextage-open/bin/startNextageOpen.sh        is missing.
 ** /opt/nextage-open/bin/diagnosisServer        is missing.
 ** /opt/nextage-open/include/nextage-open.hpp       is missing.
 ** /opt/nextage-open/bin/NxOpenCore         is missing.
 ** /opt/nextage-open/bin/unlock_iob         is missing.

---

                              *******
Done all test, Result is ...  False
                              *******
    Installability Checker version = 1.0.21
    For any issue please report to TORK or https://github.com/start-jsk/rtmros_hironx/issues

---

Saving results.... to robot-system-check-result-localhost-20140708-230732.zip}
Send following file to maintainers
  robot-system-check-result-localhost-20140708-230732.zip

Shared connection to qnx closed.
Exit request sent.
0
;;
;;
;; Done check scripts, please check robot-system-check-qnx.log file
;;
;;

;; RESULT: SUCEEDED
hit any key to exit
```

**Comment by 130s** [comment]
Storing the established ssh connection works great to minimize the number of password input.

**Comment by k-okada** [comment]
fixed typo, and https://github.com/start-jsk/rtmros_hironx/pull/204#issuecomment-51675540 is bug

**Comment by k-okada** [comment]
need to merge https://github.com/k-okada/rtmros_hironx/commit/6e2b24edc0a814a777d761c053c688df6c6bcc9f, and 
https://github.com/k-okada/rtmros_hironx/commit/73917567c4d40b6f27428acf49c1a220865cc327,

updated `store_ssh_return_exit` branch with above commits, please check again

**Pull request merged**